HACKING

```diff
@@ -20,7 +20,7 @@ Checkout the source code by cloning it from GitHub
 If you want to have a personal fork, in order to commit occasional patches,
 fork it on GitHub then clone your fork
 
-    git clone git@github.com:<username>/GCompris-qt.git
```

Comment:
this works if you have a ssh key (and this allows you to push without inputting your account/password everytime)